applications/dashboard/views/modules/cropimage.php

```diff
@@ -1,4 +1,4 @@
-<div class="padded">
+<div class="padded thumbnail-message">
```

Comment:
This is a bit too generic of a name. `change-picture-instructions` is clearer, since we may have thumbnails in multiple contexts.